packages/agents/relayer/src/bindings/relays/index.ts

```diff
@@ -113,32 +112,37 @@ export const pollCache = async () => {
           gasPrice: gasPrice.toString(),
         });
 
-        const gasLimit = await txservice.getGasEstimate(+domain, transaction);
+        //const gasLimit = await txservice.getGasEstimate(+domain, transaction);
+        const gasLimit = await rpcProvider.estimateGas(transaction);
```

Comment:
won't we run into more rpc errors without going through the txservice?

Reply:
I believe that the latest version of ethersjs manages most of them, but we need to check txservice deeply later. for now current changes will make x1-testnet work properly as I tested

Reply:
all good to merge? @LayneHaber